supporting_content/dimensions/skupriceid.md

```diff
@@ -0,0 +1,22 @@
+## Example provider mappings 
+
+Current column mappings found in available data sets:
+
+| **Provider** | **Data set**                | **Column**                             |
+| ------------ | --------------------------- | -------------------------------------- |
+| AWS          | CUR                         | pricing/rate\_code \| pricing/rate\_id |
+| Azure        | Cost details export or API  | ProductID                              |
```

Comment:
I do not see a ProductID column in my sample Azure cost data (Enterprise Agreement).
There is a ProductName column, but it does not identify the price used for the charge.
For example, a ProductName "Virtual Machines F/FS Series - F8/F8s - EU North" may be charged at on-demand rate or at reservation rate.

Reply:
The Azure column compositions are unfortunately inconsistent from one account type to the next.  Per [this documentation](https://learn.microsoft.com/en-us/azure/cost-management-billing/automate/understand-usage-details-fields), it's only available in MCA agreements.  I'm not sure if we need to specify that detail here or not...